src/chrome/content/rules/Css3test.com.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="Css3test.com">
+	<target host="css3test.com" />
+	<target host="www.css3test.com" />
```

Comment:
This target has an invalid certificate (`www.github.com`) but can be redirected to https://css3test.com/